public/modules/dashboards/controllers/fbf.client.controller.js

```diff
@@ -1373,6 +1377,26 @@ angular.module("dashboards").controller("FbfController", [
 
       $scope.copyToClipboard = shareService.copyToClipboard;
 
+      $scope.export_pdf = function() {
+        // TO DO
+      };
+
+      $scope.signup_mailing = function() {
+        $("#MailChimpModal").modal("show");
+      };
+      $scope.close_signup = function() {
+        $("#MailChimpModal").modal("hide");
+      };
+
+      $scope.join_whatsapp = function() {
+        $scope.whatsapp_link =
+          "https://chat.whatsapp.com/FH2hls1iOeNJF844L4cOjr";
```

Comment:
We should make values like this configurable 'from outside'.
So the same code can be used for multiple countries/groups/situations etc..

Reply:
Thanks @elwinschmitz! went through these comments and made adjustments :)